### ISSUE TITLE
LDAP integration for user authentication

### ISSUE THREAD

**Issue by Zorlin** [DamageLabs/kandan#420]
Hey there,

We use LDAP/Active Directory for everything (well, LDAP, really, but both would be nice) and if we wanted to use this in our wider organization instead of just for our team we would need LDAP integration.

Any thoughts on adding it? I might eventually start trying to hack it in if it isn't a priority for anyone else.

EDIT: It appears Omniauth will let me put this in with very little work, so it's sort of already there. I'll have to test it out and see how well it works, but this is _sort of_ solved. I would like there to be official documentation on adding/using LDAP support and eventually some sort of group integration, however.

**Comment by bodrovis** [comment]
https://github.com/intridea/omniauth-ldap

**Comment by Zorlin** [comment]
Yes, Omniauth-LDAP would be an excellent way to implement this and is what tons of projects use. It's my first choice if I end up needing to implement it.

I don't think kandan as it currently exists would allow and end user to add LDAP easily, however. I'm a systems adminstrator, not (yet) a Ruby programmer :)

**Comment by scouttyg** [comment]
@bodrovis might be the best person to answer this, because to be honest I have no experience implementing with LDAP. That being said, I wouldn't mind a configurable plugin to Kandan that adds this in, but I'm not sure if we should include it by default.

**Comment by 0vermind** [comment]
+1

**Comment by fusion94** [comment]
Closing as part of Kandan revival project. The app is being rebuilt from scratch on a modern stack (Angular + Express/Node). See the upcoming kandan-core repository for the new version.
